build.gradle.kts

```diff
@@ -25,7 +25,7 @@ plugins {
     // Kotlin support
     id("org.jetbrains.kotlin.jvm") version "1.5.30"
     // Gradle IntelliJ Plugin
-    id("org.jetbrains.intellij") version "1.2.1"
+    id("org.jetbrains.intellij") version "1.5.2"
```

Comment:
I'm not sure what this means for users on slightly older versions of intellij. Doesn't it break compatibility?